jnagmp/src/main/java/com/squareup/jnagmp/LibGmp.java

```diff
@@ -324,8 +324,19 @@ public static void __gmpz_export(Pointer rop, Pointer countp, int order, int siz
   public static native int __gmpz_invert(mpz_t rop, mpz_t op1, mpz_t op2);
 
   /**
-   * Calculate the Legendre symbol (a/p). This is defined only for p an odd positive prime, and
-   * for such p it’s identical to the Jacobi symbol.
+   * Calculate the Jacobi symbol (a/b). This is defined only for b odd.
+   */
+  public static native int __gmpz_jacobi(mpz_t a, mpz_t p);
+
+  /**
+   * Calculate the Jacobi symbol (a/b) with the Kronecker extension (a/2)=(2/a)
+   * when a odd, or (a/2)=0 when a even.
+   */
+  public static native int __gmpz_kronecker_si(mpz_t a, long p);
```

Comment:
There's a mismatch between the doc and param names here, "b" vs. "p".  I notice even the GMP docs are slightly inconsistent, using P for legendre and B for jacobi and kronecker.  Maybe we should follow the wikipedia nomenclature and use P for legendre and jacobi and N for kronecker?